### PR TITLE
Added some tests for settings->password, added new getters and method…

### PR DESCRIPTION
**Pull request by somalil**
…s in portal.page.js and settingsPage.js

### CODE REVIEW

File: test/pageobjects/portal/settingsPage.js [hunk 1]
```diff
@@ -16,6 +17,10 @@ class SettingsPage extends PortalPage{
         this.fullName.getAttribute('title')
     }
 
+    goToPasswordTab(){
+        this.passwordTab.click();
+    }
+
 }
 
 export default new SettingsPage();
```
Comment (michael120288) [inline test/pageobjects/portal/settingsPage.js:26]: Looks very great, but you need to add one more line)

File: test/specs/dropDownMenu/settings/password.spec.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+describe('FUNCTIONAL', () => {
+
+    before( () => {
+        browser.login(roleNew.email, roleNew.password);
+        MainPage.goToSettings();
+        SettingsPage.goToPasswordTab();
+    })
+
+    xdescribe('Toggle visibility icon functional', () => {
```
Comment (michael120288) [inline test/specs/dropDownMenu/settings/password.spec.js:14]: is it a mistake or it is suppose to be like this? (xdescribe =  describe)

**Review by michael120288 — CHANGES_REQUESTED** [review]
just couple adjustment need

File: test/specs/dropDownMenu/settings/password.spec.js (hunk 2 above)
Comment (somalil) [reply test/specs/dropDownMenu/settings/password.spec.js:14]: xdescribe = describe.skip

**Comment by somalil** [comment]
Changes added, please approve.

**Review by michael120288 — APPROVED** [review]

**Pull request merged**